bin/ch/ChakraRtInterface.cpp

```diff
@@ -138,13 +133,15 @@ bool ChakraRTInterface::LoadChakraDll(ArgInfo* argInfo, HINSTANCE *outLibrary)
     m_jsApiHooks.pfJsrtDiagGetObjectFromHandle = (JsAPIHooks::JsrtDiagGetObjectFromHandle)GetChakraCoreSymbol(library, "JsDiagGetObjectFromHandle");
 
 #ifdef _WIN32
```

Comment:
Does this still need to be under Win32? Presumably any utf8 APIs are xplat, not win32 specific